art/attacks/evasion/adversarial_patch/adversarial_patch_pytorch.py

```diff
@@ -475,13 +483,13 @@ def _random_overlay(
         return patched_images
 
     def generate(  # type: ignore
-        self, x: np.ndarray, y: np.ndarray | None = None, **kwargs
+        self, x: np.ndarray, y: np.ndarray | list[dict[str, np.ndarray | "torch.Tensor"]] | None = None, **kwargs
     ) -> tuple[np.ndarray, np.ndarray]:
         """
         Generate an adversarial patch and return the patch and its mask in arrays.
 
         :param x: An array with the original input images of shape NCHW or input videos of shape NFCHW.
-        :param y: An array with the original true labels.
+        :param y: The true or target labels.
```

Comment:
A minor suggestion to include format of expected data in the dict (e.g. boxes, scores, labels)? Torchvision models conform to this format, but users who may be more familiar with YOLO may feed targets here that are not in the correct format?

Reply:
@kieranfraser Thank you, that's good point!